src/blocks/modal.css

```diff
@@ -1,178 +1,206 @@
 .modal {
+  display: flex;
   position: fixed;
   top: 0;
   left: 0;
   bottom: 0;
   right: 0;
   background-color: rgba(0, 0, 0, 0.5);
-  z-index: 99;
-  align-items: center;
-  justify-content: center;
-  display: flex;
-  visibility: hidden;
+  z-index: 999;
+  padding: 0 19px;
+  visibility: none;
+  pointer-events: none;
   opacity: 0;
-  transition: visibility 0s linear 0.5s, opacity 0.5s linear;
+  transition: visibility 0s 0.6s, opacity 0.6s linear;
 }
 
 .modal_opened {
-  display: flex;
   visibility: visible;
+  pointer-events: all;
   opacity: 1;
-  transition-delay: 0s;
+  transition: visibility 0s, opacity 0.6s linear;
```

Comment:
Needs correction: the avatar form width should not change when you enter something to the input

Reply:
is the reason you highlight line 21 the line I need to fix?

Reply:
Maybe, you should find the line yourself